apps/pdc-frontend/src/middleware.ts

```diff
@@ -26,7 +26,7 @@ const cspBase = {
   'form-action': [SELF],
   'frame-ancestors': [NONE],
   'worker-src': [BLOB],
-  'connect-src': [SELF, getOpenFormsHost(), 'https://virtuele-gemeente-assistent.nl', DATA, BLOB],
+  'connect-src': [SELF, getOpenFormsHost(), 'wss:', 'ws:', 'https://virtuele-gemeente-assistent.nl', DATA, BLOB],
```

Comment:
Dit is niet goed, ik zou alleen `wss:` willen toestaan en niet `ws:` (niet secure) en ik wil de domeinnamen noemen.